samples/tls_skip_cert_verification.js

```diff
@@ -0,0 +1,13 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export let options = {
+    // When this option is enabled (set to true), mismatches in hostname
+    // between target system and TLS certificate will be ignored
```

Comment:
It will also trust certificates with unknown issuers or downright misissued certificates.